src/nxdoc/nuxeo-tools/nuxeo-dev-tools-extension.md

```diff
@@ -73,7 +73,7 @@ By using hot reload, the modifications done on your Studio project will be visib
 
 1. Install the extension, as explained in the section [Building from GitHub](#building-from-github)
 1. In `/bin/nuxeo.conf`, make sure the Dev Mode is enable.</br>
-    If not, stop your server, modify the file and save it.
+    If not, stop your server, modify the file by adding the `org.nuxeo.dev=true` line and save it.
```

Comment:
Can't we rather mention `./bin/nuxeoctl config` to get, stop and set the wanted value?

I'm not sure about the context: were are we supposed to be located in the filesystem? Is it assumed that we're in `NUXEO_HOME` or that we know where it is and can directly refer to `nuxeoctl` or `./bin/nuxeoctl` (same for the default `./bin/nuxeo.conf`) ?

Note: unless specific, `/bin/nuxeoctl` is not a valid path.